core/src/main/java/com/amplifyframework/predictions/models/VoiceType.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Contains details about a specific voice to be
+ * used for synthesizing speech.
+ */
+public interface VoiceType {
```

Comment:
Should this have both a `getVoiceName()` and `getLanguageCode()`, to mirror the values that are in the `AWSVoiceType`?

Reply:
iOS's core [`VoiceType`](https://github.com/aws-amplify/amplify-ios/blob/master/Amplify/Categories/Predictions/Models/VoiceType.swift) doesn't contain anything about language. Strangely enough, even the [plugin's extension](https://github.com/aws-amplify/amplify-ios/blob/cac730cc218edd7718c5bb4640975545fb6d0d2a/AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/VoiceType%2BAWSExtension.swift) doesn't seem to contain language code (outside of the constant name that is), which means that they won't be able to support bilingual TTS feature.

Reply:
Since it wasn't clear whether the intention was to completely remove language from the equation or if they wanted to keep core `VoiceType` as generic as possible, I decided to keep the core `VoiceType` interface match up with iOS and only introduce the language code in the plugin specific voice type.

This allows us to support bilingual voices by separating `AWSVoiceType#HINDI_ADITI` and `AWSVoiceType#INDIAN_ENGLISH_ADITI`